### PR TITLE
Permission seeder user path

### PR DESCRIPTION
**Pull request by CybertronianKelvin**
### Changes Made

- Fixed the user path and query in RolesAndPermissionsSeeder to ensure accurate user assignment for the 'super-admin' role.

### Changelog
- Updated the user path and query to correctly locate the user for assigning the 'super-admin' role.
- No other changes were made; the fix addresses a specific issue with the user path and query in the seeder.

### CODE REVIEW

**Review by leandrogehlen — COMMENTED** [review]
I don't know `Contributors.md`

Can you setup `https://github.com/all-contributors/all-contributors` for the repository?

**Comment by CybertronianKelvin** [comment]
Hi @leandrogehlen  thanks for the feedback, I have removed the Contributors.md. I will open a new PR today and with the setup of all-contributors.

**Comment by CybertronianKelvin** [comment]
@all-contributors please add @CybertronianKelvin

**Comment by CybertronianKelvin** [comment]
Hi @leandrogehlen have install all-contributors app into my fork and updated the README.md with it. However I reckon you need to install the app on your end as you are the repo owner.

File: README.md [hunk 1]
```diff
@@ -333,8 +331,16 @@ class Role extends BaseRole
 
 ```
 
+## Contributors
```
Comment (leandrogehlen) [inline README.md:334]: Please, remove the contributors this PR.

**Comment by leandrogehlen** [comment]
@CybertronianKelvin, thanks for the answer.
I'll setup the [all contributors](https://github.com/all-contributors/all-contributors) in another PR.

**Comment by CybertronianKelvin** [comment]
No worries, let me know if there is anything else you need doing on it so you are happy to merge please.

**Comment by leandrogehlen** [comment]
The PR was merged, thanks for the contribution

**Comment by CybertronianKelvin** [comment]
My pleasure and thanks for your patience and communication.

**Pull request merged**